ament_tools/build_types/ament_cmake.py

```diff
@@ -153,11 +153,21 @@ def on_test(self, context):
         assert context.build_tests
         # Figure out if there is a setup file to source
         prefix = self._get_command_prefix('test', context)
-        if has_make_target(context.build_space, 'test') or context.dry_run:
-            yield BuildAction(prefix + [MAKE_EXECUTABLE, 'test'])
+        if not IS_WINDOWS:
+            if has_make_target(context.build_space, 'test') or context.dry_run:
+                yield BuildAction(prefix + [MAKE_EXECUTABLE, 'test'])
+            else:
+                self.warn("Could not run test for ament_cmake package because it "
+                          "has no 'test' target")
```

Comment:
`run test` should be plural and I would suggest to quote the build type:

    self.warn("Could not run tests for 'ament_cmake' package because it "
              "has no 'test' target")

It could also following the new 100 chars wrapping rule instead of at 80 chars. (Same below)